packages/scheduler/src/forks/SchedulerHostConfig.default.js

```diff
@@ -160,8 +160,38 @@ if (
   let activeFrameTime = 33;
   let fpsLocked = false;
 
+  // TODO: Make this configurable
+  // TODO: Adjust this based on priority?
+  let maxFrameLength = 150;
```

Comment:
Let’s start a bit higher to get good signal on this. 300? Enough time to get an I/O event in that restarts React.